articles/logic-apps/error-exception-handling.md

```diff
@@ -334,6 +268,37 @@ For limits on scopes, see [Limits and config](logic-apps-limits-and-config.md).
 
 <a name="get-results-from-failures"></a>
```

Comment:
This anchor tag needs to appear preceding the section "### Get context and results for failures".